e2e/dractions/disapp.go

```diff
@@ -62,6 +76,25 @@ func EnableProtectionDiscoveredApps(ctx types.TestContext) error {
 	return nil
 }
 
+func getRecipeRef(recipeType, name, appNS, mgmtNS string) *ramen.RecipeRef {
+	var recipeRef *ramen.RecipeRef
+
+	switch recipeType {
+	case "generate", "workload":
+		recipeRef = &ramen.RecipeRef{
+			Name:      name + "-recipe",
+			Namespace: appNS,
+		}
+	case "vm":
+		recipeRef = &ramen.RecipeRef{
+			Name:      "vm-recipe",
+			Namespace: mgmtNS,
```

Comment:
Don't use such names, we call this managementNamespace everywhere, we want to use the same names for the same thing everywhere in the same project.